src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public Bulldozer(String name,int year,String colour) {
```

Comment:
```suggestion
    public Bulldozer(String name, int year, String colour) {
```
where do you use color?

Reply:
do not understand .i add colour characteristic to bUldozer,track and excavator

Reply:
done